apps/fabric-website/scripts/createFlightConfig.js

```diff
@@ -48,7 +48,7 @@ module.exports.createPublicFlightConfigTask = function() {
 
     let configData = {
       version: process.env.BUILD_BUILDNUMBER || '0',
-      baseCDNUrl: path.join(argv().baseCDNUrl, process.env.BUILD_BUILDNUMBER) + '/',
+      baseCDNUrl: argv().baseCDNUrl,
```

Comment:
It seems safer to continue appending the build number here, just do it manually not with path.join().

Reply:
I want to have symmetry with the other baseCDNUrl option for internal manifest generator. Will just have this be provided by the outside yaml.